core/sup_tree_core/system_metrics_reporter.ex

```diff
@@ -20,7 +20,9 @@ defmodule AntikytheraCore.SystemMetricsReporter do
   alias Antikythera.ExecutorPool.Id, as: EPoolId
   alias AntikytheraCore.MetricsUploader
   alias AntikytheraCore.Vm
+  require AntikytheraCore.Logger, as: L
 
+  @log_message_size 100_000
```

Comment:
I think `@log_message_queue_lenght` is better.
The reason is that you used `len` and `top_len` as the number of messages.